content/How to work with Stripe.md

```diff
@@ -0,0 +1,349 @@
+# **Online payments done SIMPLE - How to work with Stripe**
+
+![main image](images/payment.jpg)
+
+Here at [Rootstrap](https://www.rootstrap.com/mobile-app-development-los-angeles/), we care about not re inventing the wheel and making things simple (along with high quality of course).
+
+In this blog post, you’ll learn how start working with stripe to add in a fast way online payments to your apps.
+
+
+## 1) WHY STRIPE?
+
+Pros
+
+ - Easy to implement and use
+
+ - Fast to implement, so your client will be happy
+
+ - Solves most of your usual payment problems, so you don't lose time or clients (even worst)
+
+ - Amazing dashboard with lot of capabilities so your client financial team can work along with you
+
+Cons
+
+ - Expensive (high % fee)
+
+## 2) INSTALLATION
+
+This post assumes you already created a stripe account and so you have access to dashboard and it configuration.
+
+#### RAILS
+
+- Add this two gems
+    - [Stripe](https://github.com/stripe/stripe-ruby) to achieve the integration
+    - [Stripe Testing](https://github.com/stripe-ruby-mock/stripe-ruby-mock) to test your integration, you don't want to end up writting lots of mocking classes, right?
+- Configure your keys & version from the stripe dashboard
+
+``` ruby
+# config/intializers/stripe.rb
+
+Rails.configuration.stripe = {
+  publishable_key: ENV['STRIPE_PUBLISHABLE_KEY'],
+  secret_key: ENV['STRIPE_SECRET_KEY']
+}
+
+Stripe.api_key = Rails.configuration.stripe[:secret_key]
+
+```
+
+#### REACT
+
+- Add this package [Stripe](https://github.com/stripe/react-stripe-elements)
+- Configure your App to use the same api key as for rails (make sure it is the same, as you start moving between envs you may forget it)
+
+Add your Stripe wrapper
+
+``` js
```

Comment:
I see there are a lot of code examples here, do you think you could put them on gists and reference them here instead of having the code in this file?

Reply:
It's pretty straightforward, you only have to create one under your user, copy and paste this code and put the link here.